publish-all-npm-packages.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+export PACKAGES="aws-cache-helpers"
+for package in $PACKAGES
+do
+  echo "Publishing packages with version $MOMENTO_CODE_REVISION_ID"
+  pushd $package
+    mv package.json package.json.ORIG
+    cat package.json.ORIG|jq ". += {\"version\": \"$MOMENTO_CODE_REVISION_ID\"}" > package.json
+    npm publish
```

Comment:
just to confirm - the package name that shows up on npmjs will be the one in package.json, right?  Not the directory name  here?

Reply:
correct. I was noodling on this a bit... do u think this bad? Just seemed weird putting POC in folder since it was in repo name also?

Reply:
yeah.  i mostly only care about what the package name looks like on npmjs.  Having the dir name and the package name match might be mildly informative but i agree it's also repetitive so i'm fine w/this approach.

Reply:
I think w/ updated namespace this works/makes sense.